traefik/values.yaml

```diff
@@ -626,22 +626,20 @@ ports:
     # -- You SHOULD NOT expose the traefik port on production deployments.
     # If you want to access it from outside your cluster,
     # use `kubectl port-forward` or create a secure ingress
-    expose: false
+    expose:
+      default: false
```

Comment:
mmh :thinking: I understand why you name it `default`, but I'm unsure about it. 
Just looking at _values_, it feels like "would you like to expose it _by default_ ?"

Wdyt about :
Option a:

```yaml
expose:
  external: true|false
```

Option b:

```yaml
expose:
  service: true|false
```

Option c:

```yaml
expose:
  default-service: true|false
```

Reply:
It comes from our previous discussions. It don't surprises me much as `expose` is now a map of `service_name: boolean`

Reply:
Expose already was a boolean tied to service, so assuming that "service" is implicit context I think `default` makes sense, although `default-service` looks fine to me as well. Moreover, exposing it by "default" is also correct for "basic" chart usage, as that's what the default service does.

I'd argue against `external`, given that that would somehow dictate how one should use the "default" service with this chart, while one can easily use the "default" service as an internal service or have multiple external services, so it seems a bit too narrow for me.

Reply:
Ok, It works for me.